spec/marten/conf/global_settings_spec.cr

```diff
@@ -245,6 +245,39 @@ describe Marten::Conf::GlobalSettings do
     end
   end
 
+  describe "#loaders" do
+    after_each do
+      Marten.setup_templates
+    end
+
+    it "setup templates uses no loaders when `templates.loaders` setting is empty array" do
+      with_overridden_setting("templates.loaders", [] of Marten::Template::Loader::Base, nilable: true) do
+        Marten.setup_templates
+        Marten.templates.loaders.size.should eq 0
+      end
+    end
+
+    it "setup templates uses configured loaders" do
+      file_system_loader = Marten::Template::Loader::FileSystem.new("/root/templates")
+      with_overridden_setting(
+        "templates.loaders",
+        [file_system_loader] of Marten::Template::Loader::Base,
+        nilable: true
+      ) do
+        Marten.setup_templates
+        Marten.templates.loaders.size.should eq 1
+        Marten.templates.loaders[0].should eq file_system_loader
+      end
+    end
+
+    it "setup templates uses default loaders when setting is nil" do
+      with_overridden_setting("templates.loaders", nil, nilable: true) do
+        Marten.setup_templates
+        Marten.templates.loaders.size.should eq 2
+      end
+    end
+  end
```

Comment:
Can we move these specs to the `spec/marten_spec.cr` file? The reason I am asking this is that the `Marten#setup` or `Marten#setup_templates` methods are part of the top-level `Marten` module, and as such the corresponding specs should live in the associated spec file. 🙂